recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+proj:
+    - '7.2.0'
```

Comment:
`7.2.0` or `7.2.1`?

Reply:
Good question. I went with the one from the [previous pin](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L602). I guess we skipped 7.2.1. For now I would keep 7.2.0 to maintain compatibility with the rest of the stack.

Reply:
Makes sense 👍

Reply:
Yes, I agree.